packages/arch/packages/navbar/src/PrimaryNav.js

```diff
@@ -72,7 +71,7 @@ export const PrimaryNavItem = styled(ItemElement)(({ depth, isSelected }) => {
   return {
     border: 0,
     borderRight: '1px solid transparent',
-    color: colors.N90,
+    color: isSelected ? colors.N90 : mouseIsOverNav ? colors.N70 : colors.N40,
```

Comment:
What's the contrast like on this? Does it hit AA accessibility?

Reply:
Almost certainly not, but I haven't changed the experience at all when your cursor is over the nav (only when it's inactive and shouldn't distract from the content)

Also I made the text bigger so that's a win 🙂